lighthouserc.json

```diff
@@ -26,7 +26,7 @@
           {
             "matchingUrlPattern": ".*",
             "assertions": {
-              "categories:performance": ["warn", { "aggregationMethod": "median-run", "minScore": 0.8 }],
+              "categories:performance": ["error", { "aggregationMethod": "median-run", "minScore": 0.54 }],
```

Comment:
I think this is the wrong performance assertion? I see in the latest run that it failed on the category pages, but not on the homepage where the score was 0.55.